File: apps/s-2-admin/src/api/appModel/AppModelWhereInput.ts
import { StringFilter } from "../../util/StringFilter";
import { JsonFilter } from "../../util/JsonFilter";
import { DateTimeFilter } from "../../util/DateTimeFilter";
import { CredentialListRelationFilter } from "../credential/CredentialListRelationFilter";
import { WebhookListRelationFilter } from "../webhook/WebhookListRelationFilter";
import { ApiKeyListRelationFilter } from "../apiKey/ApiKeyListRelationFilter";

export type AppModelWhereInput = {
  id?: StringFilter;
  dirName?: StringFilter;
  keys?: JsonFilter;
  createdAt?: DateTimeFilter;
  updatedAt?: DateTimeFilter;
  credentials?: CredentialListRelationFilter;
  webhook?: WebhookListRelationFilter;
  apiKey?: ApiKeyListRelationFilter;
};

File: apps/cactus-demo-dotnet/src/APIs/AppModel/Dtos/AppModelWhereInput.cs
using CactusDemoDotnet.Core.Enums;

namespace CactusDemoDotnet.APIs.Dtos;

public class AppModelWhereInput
{
    public string? Id { get; set; }

    public string? DirName { get; set; }

    public string? Keys { get; set; }

    public CategoriesEnum? Categories { get; set; }

    public DateTime? CreatedAt { get; set; }

    public DateTime? UpdatedAt { get; set; }

    public List<int>? Credentials { get; set; }

    public List<string>? Webhook { get; set; }

    public List<string>? ApiKey { get; set; }
}
